class-00/lab-a/let-const/js/app.js

```diff
@@ -36,12 +36,12 @@ function displayPics(){
     }
   }
   console.log(rando);
-  // TODO: In a sentence or two, explain why the previous line of code threw an error when we changed the variable declaration from `var to `let`.
+  // TODO: In a sentence or two, explain why the previous line of code threw an error when we changed the constiable declaration from `const to `let`.
   // PUT YOUR RESPONSE IN THIS COMMENT
```

Comment:
Consider a response to this question. If you are having trouble understanding why the error is thrown read through some of the resources or ask a TA for help.